warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java

```diff
@@ -155,21 +156,33 @@ public boolean apply(Key key) {
             return apply(key, true);
         }
 
+        public boolean apply(Key key, String fieldName, String fieldValue) {
+            return apply(key, true, fieldName, fieldValue);
+        }
+
         @Override
         public boolean peek(Key key) {
             return apply(key, false);
         }
 
-        private boolean apply(Key key, boolean update) {
+        public boolean peek(Key key, String fieldName, String fieldValue) {
+            return apply(key, false, fieldName, fieldValue);
+        }
+
+        public boolean apply(Key key, boolean update) {
             final DatawaveKey datawaveKey = new DatawaveKey(key);
-            final String keyFieldName = JexlASTHelper.deconstructIdentifier(datawaveKey.getFieldName(), false);
+            final String FieldName = JexlASTHelper.deconstructIdentifier(datawaveKey.getFieldName(), false);
```

Comment:
nit: fieldName would be a more consistent way to name this variable

Reply:
fixed